test/config/integration/server_cors_filter.json

```diff
@@ -0,0 +1,101 @@
+{
+  "listeners": [
+  {
+    "address": "tcp://{{ ip_loopback_address }}:0",
+    "filters": [
+    {
+      "type": "read",
+      "name": "http_connection_manager",
+      "config": {
+        "codec_type": "http1",
+        "access_log": [
```

Comment:
I think you should be able to remove the access log config from this clone.  Mind checking?

Reply:
I'll give it a try.